Source/Core/Core/HW/WiimoteEmu/Attachment/Attachment.cpp

```diff
@@ -50,3 +50,14 @@ void ControllerEmu::Extension::GetState(u8* const data)
 {
 	((WiimoteEmu::Attachment*)attachments[active_extension].get())->GetState(data);
 }
+
+bool ControllerEmu::Extension::IsButtonPressed()
+{
+	// Since we want to use this to know if disconnected Wiimotes want to be connected, and disconnected
+	// Wiimotes (can? always?) have their active_extension set to -1, we also have to check the switch_extension
+	if (active_extension > 0)
```

Comment:
0 is used to signal that no extension is connected. I'm not sure what exactly -1 means, though. I guess it's "not initialized yet" or something along those lines?

https://github.com/dolphin-emu/dolphin/blob/master/Source/Core/Core/HW/WiimoteEmu/WiimoteEmu.cpp#L211